client/src/components/layouts/default.tsx

```diff
@@ -8,9 +8,9 @@ import { createSelector } from 'reselect';
 import latoBoldURL from '../../../static/fonts/lato/Lato-Bold.woff';
 import latoLightURL from '../../../static/fonts/lato/Lato-Light.woff';
 import latoRegularURL from '../../../static/fonts/lato/Lato-Regular.woff';
-import robotoBoldURL from '../../../static/fonts/roboto-mono/RobotoMono-Bold.woff';
-import robotoItalicURL from '../../../static/fonts/roboto-mono/RobotoMono-Italic.woff';
-import robotoRegularURL from '../../../static/fonts/roboto-mono/RobotoMono-Regular.woff';
+import hackZeroSlashBoldURL from '../../../static/fonts/hack-zeroslash/Hack-ZeroSlash-Bold.woff';
+import hackZeroSlashItalicURL from '../../../static/fonts/hack-zeroslash/Hack-ZeroSlash-Italic.woff';
+import hackZeroSlashRegularURL from '../../../static/fonts/hack-zeroslash/Hack-ZeroSlash-Regular.woff';
```

Comment:
Also, why are only the woff files included here and not the woff2 files?

Reply:
I am trying to include the woff2 files but am getting the error `Cannot find module '../../../static/fonts/hack-zeroslash/Hack-ZeroSlash-Regular.woff2' or its corresponding type declarations.` Would it be okay to leave them out for now, as the other fonts only have `woff`?

Reply:
Correct me if I'm wrong, but the reason we are importing these font files here is so that we can preload them. Since woff2 is now preferred over woff and is supported in all browsers except IE then I think if we are going to preload these fonts it should be the woff2 files, not the woff files. By the way, it appears the main difference between woff and woff2 is that the later has a better compression algorithm, so we'll save even more bandwidth by preloading woff2 instead.

Reply:
I hear you that woff2 is more efficient. I am just having trouble including the file path. Do you know why the error above might be happening? Sorry, I'm a bit new to front-end programming.

Reply:
Hmm... when I add a `2` to the end of these file paths (and to the `type` attributes on the preload `link` tags below) I don't get any errors and I can see all four woff2 files loading in the network tab of dev tools. I'm not sure why that would work for me but error for you? Perhaps @ieahleen or @ahmadabdolsaheb might have some insight?

Reply:
Not sure why that might be happening. We could move forward with the current spec if woff2 is giving you a hard time and we could create a separate issue for woff --> woff2 for all fonts.

Additionally, not sure if we are using hackZeroSlashBoldItalicURL or hackZeroSlashItalicURL. Feel free to drop them if they are not being used.

Reply:
Hmm maybe it has to do with my local dev environment. Would it be okay to create the separate pull request? I've added a commit to remove the BoldItalic font, as I don't think it is used.

Reply:
I'm fine with leaving this PR preloading the woff files and opening a separate PR to switch to woff2.